lib/streamlit/elements/markdown.py

```diff
@@ -82,11 +82,14 @@ def markdown(
 
     @gather_metrics("code")
     def code(
-        self, body: SupportsStr, language: Optional[str] = "python"
+        self,
+        body: SupportsStr,
+        language: Optional[str] = "python",
+        line_numbers: bool = False,
     ) -> "DeltaGenerator":
         """Display a code block with optional syntax highlighting.
 
-        (This is a convenience wrapper around `st.markdown()`)
+        (This is a convenience wrapper around ``st.markdown()``)
```

Comment:
Do we need double `` here?

Reply:
Yeah, the docstrings contain reStructuredText, not markdown. Inline code in reST should be enclosed within two backticks.

Reply:
Else, text within single backticks are rendered as italics in the docs.